### ISSUE TITLE
readOnly property not working

### ISSUE THREAD

**Issue by dnepro** [airbnb/react-dates#660]
I tried setting the "readOnly"-property on my component but it didn't work. After trying to understand whats wrong, i've found out that the Storybook example isn't working as well.

**Comment by timhwang21** [comment]
Just to be sure, `disabled` is intended to disable the input while `readOnly` prevents typing but still allows the calendar to open.

Are you looking for `disabled`?

**Comment by dnepro** [comment]
Yes, the DatePicker should open, but the user shouldn't be able to select a new Date, which at the moment is possible with the readOnly attribute set.. in my opinion this is not correct behaviour?

**Comment by timhwang21** [comment]
I'm not 100% sure and don't want to spread misinformation, hopefully @majapw can chime in?

**Comment by astolfivincent** [comment]
@timhwang21 and @dnepro see: https://stackoverflow.com/a/57775723/5652210
